content/account-data-matching.md

```diff
@@ -1,11 +1,3 @@
----
-title: Account Data Matching
-objectives:
-- Explain the security risks associated with missing data validation checks
-- Implement data validation checks using long-form Rust
-- Implement data validation checks using Anchor constraints
----
-
```

Comment:
This looks like a big change but it's not. Made a quick script to read these and move them to the course-structure file.